pylot/drivers/carla_obstacles_driver_operator.py

```diff
@@ -0,0 +1,48 @@
+import carla
+
+import erdos
+
+import pylot.utils
+from pylot.drivers.carla_base_gnss_driver_operator import (
+    CarlaBaseGNSSDriverOperator)
+from pylot.perception.detection.obstacle import Obstacle
+from pylot.perception.messages import ObstaclesMessage
+
+
+class CarlaObstaclesDriverOperator(CarlaBaseGNSSDriverOperator):
+    """Publishes the bounding boxes of all vehicles and people retrieved from
+    the simulator at the provided frequency.
+
+    Args:
+        vehicle_id_stream: Stream on which the operator receives the ID of the
+            ego vehicle. The ID is used to get a simulator handle to the
+            vehicle.
+        obstacles_stream: Stream on which the operator sends the obstacles.
+        frequency: Rate at which the pose is published, in Hertz.
+        flags: Object used to access absl flags.
+    """
+    def __init__(self, vehicle_id_stream: erdos.ReadStream,
+                 obstacles_stream: erdos.WriteStream, frequency: float, flags):
+        transform = pylot.utils.Transform(pylot.utils.Location(),
+                                          pylot.utils.Rotation())
+        gnss_setup = pylot.drivers.sensor_setup.GNSSSetup(
+            self.config.name, transform)
+        super().__init__(vehicle_id_stream, obstacles_stream, gnss_setup,
+                         frequency, flags)
```

Comment:
Does this mean that a new `GNSS` will be installed for every operator?

Reply:
Yes